tools/setup_env.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash -e
-gcloud components install app-engine-java
-
-cd ui
-echo Installing the Angular command-line tool, to serve/build the UI.
-sudo npm install -g @angular/cli
-echo Installing UI project dependencies.
-npm install
-
-
-cd ../tools
-./download_swagger_cli.sh
-./download_cloud_sql_proxy.sh
```

Comment:
I'm not sure I follow. If we might be using cloudsql proxy, should we also leave it in setup_env?